tests/lib/cisco_test.py

```diff
@@ -364,7 +372,8 @@
         'version-2-multicast-listener-report',
     },
     'option': {'established',
-               'tcp-established'}
+               'tcp-established',
+               'fragments'}
```

Comment:
There is already an option called is-fragment. For the sake of keeping things unified can we use that?

Reply:
Although it has the same functionality, using if-fragment add more complexity to an ACL. The option "is-fragment" has to be used together with "fragment-type". This should be used when an ACL has to be configured with a deep granularity. To handle only fragments, or to be more specific non-initial fragments, the keyword "fragments" added to any ACE, makes that ACE only applicable to a non-initial fragment. Cisco provides both options, and I think Capirca could support both.

Reply:
You can see here a good explanation about how Cisco implement fragments support: https://www.cisco.com/c/en/us/td/docs/iosxr/ncs5xx/ipaddress/63x/b-ip-addresses-cg-63x-ncs5xx/b-ip-addresses-cg-63x-ncs5xx_chapter_01.pdf 

- Page 17: the use of "fragments" keyword
- Page 19: use case for fragment-type and is-fragment